GoogleDataTransport/GDTCCTTests/Unit/GDTCCTNanopbHelpersTest.m

```diff
@@ -109,10 +109,34 @@ - (void)testBytesAreDecodable {
   pb_istream_t istream =
       pb_istream_from_buffer([encodedBatchLogRequest bytes], [encodedBatchLogRequest length]);
   XCTAssertTrue(pb_decode(&istream, gdt_cct_BatchedLogRequest_fields, &decodedBatch));
-  XCTAssert(decodedBatch.log_request_count == batch.log_request_count);
-  XCTAssert(decodedBatch.log_request[0].log_event_count == batch.log_request[0].log_event_count);
-  XCTAssert(decodedBatch.log_request[0].log_event[0].event_time_ms ==
-            batch.log_request[0].log_event[0].event_time_ms);
+  XCTAssertEqual(decodedBatch.log_request_count, batch.log_request_count);
+  XCTAssertEqual(decodedBatch.log_request[0].log_event_count, batch.log_request[0].log_event_count);
+  XCTAssertEqual(decodedBatch.log_request[0].log_event[0].event_time_ms,
+                 batch.log_request[0].log_event[0].event_time_ms);
+  XCTAssertEqual(decodedBatch.log_request[0].log_event[0].event_uptime_ms,
+                 batch.log_request[0].log_event[0].event_uptime_ms);
+  pb_release(gdt_cct_BatchedLogRequest_fields, &batch);
+  pb_release(gdt_cct_BatchedLogRequest_fields, &decodedBatch);
+}
+
+- (void)testDecodedEventTimestampMatchToBatchContent {
```

Comment:
Can we have some tests directly for GDTCCTConstructLogEvent and GDTCCTConstructLogRequest API to ascertain the availability of the values for LogEvent? These are critical fields that the users of Firelog will be dependent on.